tools/fs/mkromfsimg.sh

```diff
@@ -51,6 +51,8 @@
 #
 ############################################################################
 
+source .config
```

Comment:
Did you verify this ?  When I check in my machine with your changes, i get below error.

../tools/fs/mkromfsimg.sh: 54: ../tools/fs/mkromfsimg.sh: source: not found